crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -9,11 +9,98 @@ use ruff_python_semantic::{Definition, MemberKind, SemanticModel};
 use ruff_text_size::{Ranged, TextRange};
 
 use crate::checkers::ast::Checker;
-use crate::docstrings::sections::{SectionContexts, SectionKind};
+use crate::docstrings::sections::{SectionContext, SectionContexts, SectionKind};
 use crate::docstrings::styles::SectionStyle;
 use crate::registry::Rule;
 use crate::rules::pydocstyle::settings::Convention;
 
+/// ## What it does
+/// Checks for function docstrings that do not have a returns section.
```

Comment:
I'm not entirely sure how to phrase it but should the documentation mention that it only applies to functions that have a body with a return statement?